react-app-drawer-component/src/app-drawer.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+class AppDrawer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      drawerOpen: false,
+      showMenu: true
+    };
+    this.HandleClickDrawer = this.HandleClickDrawer.bind(this);
+  }
+
+  HandleClickDrawer() {
```

Comment:
I noticed that you are defining a lot of your methods with an uppercase first letter. This is not "wrong", but it's definitely not what JS programmers do.